modbusrtu-requirements.txt

```diff
@@ -0,0 +1 @@
+minimalmodbus==1.0.2
```

Comment:
I guess that the `modbustcp` module we currently used can't be reused here, would be nice to have a one module solution to cover both cases. Maybe minimalmodbus supports tcp as well?

Reply:
Correct. Unfortunately modbus tcp and rtu are two quite different things. Also minimalmodbus does not support tcp.